index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Apple Clone</title>
+  <link rel="stylesheet" href="./assets/css/style.css">
+</head>
+<body>
+  <header>
+    <nav class="nav-container">
+      <ul class="list-inline">
+        <li class="nav-item"><a href="#" class="menu-item"><img class="apple-logo" src="./assets/images/globalnav.svg" alt="Apple logo" ></a></li>
+        <li class="nav-item"><a href="#" class="menu-item"><span>Store</span></a></li>
+        <li class="nav-item"><a href="#" class="menu-item"><span>Mac</span></a></li>
+        <li class="nav-item"><a href="#" class="menu-item"><span>ipod</span></a></li>
+        <li class="nav-item"><a href="#" class="menu-item"><span>iPhone</span></a></li>
+        <li class="nav-item"><a href="#" class="menu-item"><span>iPad</span></a></li>
+        <li class="nav-item"><a href="#" class="menu-item"><span>iTunes</span></a></li>
+        <li class="nav-item"><a href="#" class="menu-item"><span>Support</span></a></li>
+        <li>
+          <form action="#" class="search">
+            <i></i>
+            <input type="text" name="search">
+          </form>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <section class="main">
```

Comment:
[OPTIONAL] change the `<section>` tag to be the `<main>` tag

Reply:
Thank you, my team gave me the same suggestion on morning meeting, and it's already applied.